### DISCUSSION TITLE
Goal 2: End hunger, achieve food security and improved nutrition and promote sustainable agriculture

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#24] [General]
Chat about anything related to [Goal 2: End hunger, achieve food security and improved nutrition and promote sustainable agriculture](https://github.com/Project-Resilience/platform/blob/main/goals/goal_02.md)

**Comment by Steven-Birkinbine** [comment]
The WFP works to reduce world hunger: https://www.wfp.org/resilience-building
The Great Green Wall is a UN initiative to stop the Sahara Desert from expanding and returning water and food security to the people of the Sahel: https://www.unccd.int/our-work/ggwi

The Great Green Wall has already brought food security to over 500k people.

**Comment by ofrancon** [comment]
The World Food Programme has a data visualizer here: https://dataviz.vam.wfp.org/
Data is accessible, for instance here: https://dataviz.vam.wfp.org/economic/export-data?current_page=1

**Comment by ofrancon** [comment]
This paper "establishes the groundwork for a global, data-driven early warning system designed to anticipate and detect food insecurity". It might be possible to learn which actions to take to avoid food insecurity.

Forecasting trends in food security with real time data
https://doi.org/10.1038/s43247-024-01698-9

**Comment by danyoungday** [comment]
# World Food Programme Food Security Optimization

Similar to the Covid-19 NPI Optimization use-case, one can forecast food security and make prescriptions for aid based on this forecast. Forecasting has already been done by the WFP as mentioned above using reservoir computing. The paper can be found at: [https://doi.org/10.1038/s43247-024-01698-9](https://doi.org/10.1038/s43247-024-01698-9 ) and the Github repo can be found at: [https://github.com/ducciopiovani/FamPredAI](https://github.com/ducciopiovani/FamPredAI)

## Data
- The WFP data visualizer can be found here: [https://dataviz.vam.wfp.org/](https://dataviz.vam.wfp.org/)
- More specifically for context and outcomes, processed data can be found in the WFP Github.
- (Items we don't currently have are in bold)

### Outcome
- Food Consumption Score (FCS) - survey on how much of each food group was consumed in the past 7 days
- **Duplication and waste in intervention**

### Context
- Metrics
    - FCS
    - Reduced Coping Strategies Index (rCSI) - survey on measures households engage in due to food shortages in the past 7 days
- Climate
    - Rainfall, 1 month and 3 month rainfall anomaly
    - Normalized Difference Vegetation Index (NDVI), 1 month NDVI anomaly
 - Economic
    - Alert for Price Spikes (ALPS) for food prices
    - Food inflation
    - Headline inflation
    - Currency exchange unofficial
 - Conflict
     - Battle fatalities
     - Violence against civilians fatalities
     - Remote violence/explosions fatalities
 - Other
    - Seasonal calendars
    - Ramadan
    - Day of the year

### Actions
- **Food distribution**
- **Commodity pricing**

**Reply by danyoungday** [reply]
## WFP Nowcasting
[Paper](https://www.nature.com/articles/s43016-022-00587-8)
[Code](https://zenodo.org/records/6953981)

Rather than forecast autoregressively using previous FCS scores, tries to predict FCS at a single time step from just the auxiliary features

**Comment by ofrancon** [comment]
There could be some interesting information for a project here:
https://digitalgreen.org/global-impact/
```
In developing and sharing our AI-powered tools, we promote Climate Smart Agriculture (CSA) practices that are shown to have both environmental and livelihood benefits for farmers, such as optimizing use of fertilizer and water and livestock feed formulations. 
```

**Comment by danyoungday** [comment]
# AquaCrop Irrigation Strategy Optimization

The FAO has a simulator called [AquaCrop](https://www.fao.org/aquacrop/en/) which takes in lots of variables such as weather, crop type, soil type, irrigation strategy, and field management, and then outputs the yield and amount of irrigation used. A [python library version](https://aquacropos.github.io/aquacrop/) can be used as an evaluator for evolution.

## Data

### Context
- Weather
- Soil
- Crop

### Actions
- Irrigation strategy
- Field management
    - (Mulch strategy)

### Outcomes
- Yield
- Total irrigation
- Mulch percent
- ...

**Reply by danyoungday** [reply]
A demo for this project has been created [here](https://github.com/danyoungday/praise). There is also a [web-app demo](https://irrigation-strategy.streamlit.app/)

**Comment by ofrancon** [comment]
This FAO project could be interesting:
[FAO AgroAdapt Simulation Game](https://agroadapt.fao.org/): a hybrid simulation board game to tackle climate change issues on agrifood systems

3 type of players:
- Policy maker
- Community Leader
- Farmer

Plus a game master.

3 scores:
- environment
- economic
- social

15 policies to choose from for each player.

It may be possible to optimize what players should do / should have done at each turn.
It may also be possible to create an AI game master and AI players, e.g. adversarial players.

**Comment by ofrancon** [comment]
Additional resources from the [Food and Agriculture Organization of the United Nations (FAO)](https://www.fao.org/home/en):
- Data portal: [data viewer](https://gaez.fao.org/pages/data-viewer)
- PyAEZ python package: [PyAEZ](https://gaez.fao.org/pages/pyaez) Python-based Analysis of Agro-Ecological Zones
- [FAO Agro-Informatics](https://www.fao.org/agroinformatics/en): provides the digital enabling tools for FAO’s strategic technical initiatives
- FAO Agro-Informatics Platform: [FAO Hand-in-Hand Geospatial Platform](https://data.apps.fao.org/)
- FAO Digital Service Portfolio [Home | FAO Digital Services Portfolio | Food and Agriculture Organization of the United Nations](https://www.fao.org/digital-services/en)
- [FAO Risk Monitor](https://riskmonitor.fao.org/): Food security risk intelligence and early warning room
- [WaPOR](https://www.fao.org/in-action/remote-sensing-for-water-productivity/en): remote sensing for water productivity
- [SoilHave](https://soilhive.ag/app/): Open Soil Data Platform
- [WOFOST](https://www.fao.org/land-water/land/land-governance/land-resources-planning-toolbox/category/details/ru/c/1236431/): World Food Studies Simulation Model
- [EX-ACT](https://www.fao.org/in-action/epic/ex-act-tool/suite-of-tools/ex-act/en/): The Environmental eXternalities ACcounting Tool (EX-ACT, previously EX-Ante Carbon balance Tool), provides users a consistent way of estimating and tracking the outcomes of agricultural interventions on GHG emissions.
